### DISCUSSION TITLE
Unit conversions of Transporter`s Vmax

### DISCUSSION THREAD

**Discussion by Rumi0708** [Open-Systems-Pharmacology/Forum#1765] [Questions & Problems] [answered]
Dear all.
I am a Japanese university student and a PK-Sim beginner.
I have obtained information on transporter Km and Vmax from the literature.
In the literature, the unit of Vmax was stated as XX nmol/min/10^6cells. The cells used are HEK and MEL.
The Vmax units for PK-Sim were XX mol/min/kg tissue, XX mol/l/min and XX nmol/min/pmol transporter.
How should I convert XX nmol/min/10^6cells?
We would be grateful for your answer.
Yours sincerely

**Comment by StephanSchaller** [comment]
Hi @Rumi0708, A PBPK software doesn't have the info on numbers/counts or concentration of transport enzymes per transfected or 10^6 cells in specific HEK/MEL assays. So you would have to find this reference data (pmol transporter/10^6cells) in the publication (or find a general number from the literature) to translate your data.

**Reply by Rumi0708** [reply, marked as answer]
Dear @StephanSchaller .

Thank you for your reply.
I have also read previous question. #1295  
To find the number of cell/mL, the publication was presented.
It stated that 1 cm^3 = 10^9 cell number.
Does it then mean that the amount of transporters per volume should be known?
I am looking into this, but I have not found any publication yet.
I would like to know if you know of any references.
Yours sincerely.
